Firebase.RemoteConfig/source/Firebase.RemoteConfig/ApiDefinition.cs

```diff
@@ -89,20 +90,22 @@ interface RemoteConfig
 		RemoteConfig SharedInstance { get; }
 
 		// -(void)fetchWithCompletionHandler:(FIRRemoteConfigFetchCompletion _Nullable)completionHandler;
+		[Async]
 		[Export ("fetchWithCompletionHandler:")]
 		void Fetch ([NullAllowed] RemoteConfigFetchCompletionHandler completionHandler);
 
 		// -(void)fetchWithExpirationDuration:(NSTimeInterval)expirationDuration completionHandler:(FIRRemoteConfigFetchCompletion _Nullable)completionHandler;
+		[Async]
 		[Export ("fetchWithExpirationDuration:completionHandler:")]
 		void Fetch (double expirationDuration, [NullAllowed] RemoteConfigFetchCompletionHandler completionHandler);
 
 		// -(BOOL)activateFetched;
 		[Export ("activateFetched")]
 		bool ActivateFetched ();
 
-        // -(FIRRemoteConfigValue * _Nonnull)objectForKeyedSubscript:(NSString * _Nonnull)key;
-        [Export("objectForKeyedSubscript:")]
-        RemoteConfigValue ObjectForKeyedSubscript(string key);
+		// -(FIRRemoteConfigValue * _Nonnull)objectForKeyedSubscript:(NSString * _Nonnull)key;
+		[Export("objectForKeyedSubscript:")]
+		RemoteConfigValue ObjectForKeyedSubscript(string key);
```

Comment:
Nope. never expose this thing `ObjectForKeyedSubscript` or at least not named like that, you can use `GetObject` as name or you could expose a [C# indexer](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/indexers/) instead using this API internally.